src/PrestaShopBundle/Command/ShopCreatorCommand.php

```diff
@@ -0,0 +1,247 @@
+
+/**
+ * This command is used for appending the hook names in the configuration file.
+ */
+class ShopCreatorCommand extends Command
+{
+    /**
+     * @var string
+     */
+    private $dbPrefix;
+
+    public function __construct(string $dbPrefix)
+    {
+        parent::__construct();
+        $this->dbPrefix = $dbPrefix;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('prestashop:shop-creator')
+            ->addOption('orders', null, InputOption::VALUE_OPTIONAL, 'Number of orders to create', 10000)
+            ->addOption('customers', null, InputOption::VALUE_OPTIONAL, 'Number of customers without order to create', 5000)
+            ->addOption('carts', null, InputOption::VALUE_OPTIONAL, 'Number of carts to create', 50000)
+            ->addOption('cart-rules', null, InputOption::VALUE_OPTIONAL, 'Number of cart rules to create', 10000)
+            ->addOption('products', null, InputOption::VALUE_OPTIONAL, 'Number of products to create', 10000)
+            ->addOption('shopId', null, InputOption::VALUE_OPTIONAL, 'The shop identifier', 1)
+            ->addOption('shopGroupId', null, InputOption::VALUE_OPTIONAL, 'The shop group identifier', 1)
+            ->addOption('languageId', null, InputOption::VALUE_OPTIONAL, 'The languageId identifier', 1)
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        \Context::getContext()->currency = \Currency::getDefaultCurrency();
+
+        $faker = Factory::create('fr_FR');
+
+        $numberOfOrders = (int) $input->getOption('orders');
+        $numberOfCustomerWithoutOrder = (int) $input->getOption('customers');
+        $numberOfCarts = (int) $input->getOption('carts');
+        $numberOfCartRules = (int) $input->getOption('cart-rules');
+        $numberOfProducts = (int) $input->getOption('products');
+        $idLang = (int) $input->getOption('languageId');
+        $idshop = (int) $input->getOption('shopId');
+        $idShopGroup = (int) $input->getOption('shopGroupId');
+
+        $productIds = $this->getSimpleProducts($idLang);
+        for ($i = 0; $i < $numberOfCustomerWithoutOrder; ++$i) {
+            $customer = new Customer();
+            $customer->firstname = $faker->firstName;
+            $customer->lastname = $faker->lastName;
+            $customer->email = $faker->email;
+            $customer->passwd = '$2y$10$WzLnGz9j..JtTFcjfjoWr.8L/rw39NwovNRwPxf6yk/AYWcIj/1Au';
+            $customer->add();
+            $output->writeln('Customer without order ' . ($i + 1) . ' created successfully.');
+        }
+        for ($i = 0; $i < $numberOfCarts; ++$i) {
+            $cart = new Cart();
+            $cart->id_currency = 1; // Modifier cette valeur selon les devises disponibles dans votre boutique
+            $cart->add();
+
+            for ($j = 0; $j < $faker->numberBetween(1, 5); ++$j) {
```

Comment:
We should add more than five products because bigger carts are a reason for poor performance in some places. Maybe it could be configurable.